README.md

```diff
@@ -1 +1,244 @@
+
+- [ ] **[Search and filter by city]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/RileyManda/Morse Code/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+<!-- SUPPORT -->
+## <b>Show your support 🌟</b><a name="support"></a>
+
+Thank you for taking the time to explore this project! Your support means a lot to me. If you find my project valuable and would like to contribute, here is one way you can support me:
+
+ - <b>Star the project ⭐️</b>: Show your appreciation by starring this GitHub repository. It helps increase visibility and lets others know that the project is well-received.
+
+ - <b>Fork the project 🍴 🎣</b>: If you're interested in making improvements or adding new features, feel free to fork the project. You can work on your own version and even submit pull requests to suggest changes.
+
+ - <b>Share with others 🗺️</b>: Spread the word about this project. Share it on social media, mention it in relevant forums or communities, or recommend it to colleagues and friends who might find it useful.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to express my sincere gratitude to [Microverse](https://github.com/microverseinc), the dedicated reviewers, and collaborators. Your unwavering support, feedback, and collaborative efforts have played an immense role in making this journey a resounding success. I am truly grateful for your contributions and for being an integral part of my achievements.
+I would like to also send out special thanks to [Nelson Sakwa
+](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) for the original Ux design that inspired this project.
+Thank you for your continued support.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+- **Question_1**
+
+  Do I have to use the vs code specifically?
+
+  - Answer_1
+
+    You can use any code editor of your choice. <br>
+
+- **Question_2**
+
+  Where can I download node JS for installation?
+
+  - Answer_2
+
+  Node.js® is a JavaScript runtime built on Chrome's V8 JavaScript engine.
+  It can be downloaded [here](https://nodejs.org/en).
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ATTRIBUTION -->
+
+## 👥 Attribution <a name="attribution"></a>
+- This project is based on the Original design by: [Nelson Sakwa
+](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs))
+- Original logo design from: [Svgrepo](https://www.svgrepo.com/)
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+[![MIT License](https://img.shields.io/badge/License-MIT-green.svg)](./LICENSE)
+
```

Comment:
- [ ] The License link you have in your readme leads to a 404 error page. I'll suggest you add a Licence file and correctly link it in your readme. :smile: :wave: 

- - Screenshot:
![Screenshot from 2023-08-30 12-05-45](https://github.com/RileyManda/morse-code/assets/107632780/326d5324-0e3b-4b74-aa36-f4507ea877ce)